### ISSUE TITLE
catkin clean -a

### ISSUE THREAD

**Issue by Jmeyer1292** [ros-industrial/industrial_ci#30]
After merging a new Descartes pull request, I received a notification that the build had failed. A quick search reveals:

```
+++catkin clean -a

usage: catkin [-h] [-a] [--test-colors] [--version]

              [--force-color | --no-color]

              [build | clean | config | create | env | init | list | locate | profile]

              ...

catkin: error: unrecognized arguments: -a
```
See the following for the full log:
https://travis-ci.org/ros-industrial-consortium/descartes/jobs/124093593

Is there something I'm not setting? I'll keep looking, but I wanted to throw this up here in case someone else has an idea.

**Comment by 130s** [comment]
Thanks for the report. I'll look into this.
Seems related https://github.com/catkin/catkin_tools/issues/320.

**Comment by 130s** [comment]
Should be resolved by #31. @Jmeyer1292 @VictorLamoine if it isn't please let me know on this ticket.

**Comment by 130s** [comment]
You can simply re-run the Travis jobs that failed.
